app/components/profile/link_component.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Profile::LinkComponent < ViewComponent::Base
+  LINK_ICONS = {
+    github_link: 'fa-brands fa-github text-[#181717] hover:bg-[#181717]',
+    linked_in_link: 'fa-brands fa-linkedin-in text-[#0072b1] hover:bg-[#0072b1]',
+    personal_site_link: 'fa-solid fa-link hover:bg-neutral',
+    twitter_link: 'fa-brands fa-twitter text-[#26a7de] hover:bg-[#26a7de]'
```

Comment:
So I know Ariel had mentioned wanting to avoid the `[hex-code]` color syntax, which I absolutely agree is a good idea. Not sure about these 'brand colors' though as they're not part of our app's theme, so I've treated them as an exception here.

Do  you have any opinions on this @afogel ?

Reply:
this looks great! I do have some opinions haha

I think it might make sense to add css variables for the brand colors/add a theme color in the tailwind.config. My thinking is along these lines:
Not only will it be more readable/easier to understand what the colors are, but it will continue to enforce a coding style wherein we don't have one-off colors that are added. My biggest concern is leaving code examples for future developers that may be less initiated with our norms.

I think we probably could get away with using our neutrals for the github, though?